client/my-sites/upgrades/domain-search/site-redirect.jsx

```diff
@@ -60,4 +62,12 @@ var SiteRedirect = React.createClass( {
 	}
 } );
 
-module.exports = SiteRedirect;
+export default connect(
+	( state, ownProps ) => {
+		const selectedSite = ownProps.sites.getSelectedSite();
```

Comment:
Just like my previous comment above, I'd suggest that we don't introduce another instance of `sitesList.getSelectedSite()`. Then we'll also no longer need to pass `ownProps` to `mapStateToProps`.